node/src/da_block_import.rs

```diff
@@ -183,7 +178,7 @@ where
 				.map_err(|e| ClientImport(format!("Build extension fails due to: {e:?}")))?
 		},
 		_ => {
-			log::debug!(target: "DA_IMPORT_BLOCK", "V2 validation..");
+			log::debug!(target: "DA_IMPORT_BLOCK", "V2^ validation..");
```

Comment:
Is this a typo?

Reply:
Actually it was to say v2 and upward, but I réalise it's not clear.

Reply:
Oh ok